packages/@aws-cdk/aws-pipes-targets-alpha/lib/eventBridge.ts

```diff
@@ -0,0 +1,145 @@
+   * Minimum: 1
+   * Maximum: 50
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-endpointid
+   * @default none
+   */
+  readonly endpointId?: string;
+
+  /**
+   * AWS resources, identified by Amazon Resource Name (ARN), which the event primarily concerns. Any number, including zero, may be present.
+   *
+   * Minimum: 1 | 0
+   * Maximum: 1600 | 10
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-resources
+   * @default none
+   */
+  readonly resources?: string[];
+
+  /**
+   * The source of the event.
+   *
+   * Minimum: 1
+   * Maximum: 256
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-source
+   * @default none
+   */
+  readonly source?: string;
+
+  /**
+   * The time stamp of the event, per RFC3339. If no time stamp is provided, the time stamp of the PutEvents call is used.
+   *
+   * Pattern: (?=[/\.\-_A-Za-z0-9]+)((?!aws\.).*)|(\$(\.[\w/_-]+(\[(\d+|\*)\])*)*)
+   * Minimum: 1
+   * Maximum: 256
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargeteventbridgeeventbusparameters.html#cfn-pipes-pipe-pipetargeteventbridgeeventbusparameters-time
+   * @default none
+   */
+  readonly time?: string;
+}
+
+/**
+ * A EventBridge Pipes target that sends messages to an EventBridge event bus.
+ */
+export class EventBridgeTarget implements ITarget {
+  private eventBus: IEventBus;
+  private eventBridgeParameters?: EventBridgeTargetParameters;
+  public readonly targetArn: string;
+  constructor(eventBus: IEventBus, parameters?: EventBridgeTargetParameters) {
+    this.eventBus = eventBus;
+    this.targetArn = eventBus.eventBusArn;
+    this.eventBridgeParameters = parameters;
+
+    validateDetailType(parameters?.detailType);
+    validateEndpointId(parameters?.endpointId);
+    validateSource(parameters?.source);
+    validateTime(parameters?.time);
+  }
+
+  grantPush(grantee: IRole): void {
+    this.eventBus.grantPutEventsTo(grantee);
+  }
+
+  bind(pipe: IPipe): TargetConfig {
+    if (!this.eventBridgeParameters) {
+      return {
+        targetParameters: {},
+      };
+    }
+
+    return {
+      targetParameters: {
+        inputTemplate: this.eventBridgeParameters.inputTransformation?.bind(pipe).inputTemplate,
+        eventBridgeEventBusParameters: this.eventBridgeParameters,
+      },
+    };
+  }
+}
+
+function validateDetailType(detailType?: string) {
+  if (detailType !== undefined) {
+    if (detailType.length < 1 || detailType.length > 128) {
```

Comment:
Shouldn't all these max length checks be inclusive?
```suggestion
    if (detailType.length < 1 || detailType.length >= 128) {
```